Arrangement_on_surface_2/include/CGAL/IO/Fig_stream.h

```diff
@@ -1716,9 +1715,11 @@ class Fig_stream
       }
       else
       {
-        // Convert the current character to an octal string and write it.
-        sprintf (oct, "\\%03o", text[i]);
-        _ofile << oct;
+        // Convert the current character to an octal string and write
+        // it.
+        std::stringstream out;
+        out << "\\" << std::setfill('0') << std::setw(3) << std::oct << text[i];
+        _ofile << out.str();
```

Comment:
That code should stream to `_ofile` directly, instead of creating that temporary `std::stringstream`. :smile:

Reply:
I did not do that as `std::hex`  and `std::oct` change all consecutive ìnt`.